crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/staticdata/enums/stateless/StatelessOutcome.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.laa.crime.meansassessment.staticdata.enums.stateless;
+
+public enum StatelessOutcome {
+    ELIGIBLE_WITH_CONTRIBUTION,
+    ELIGIBLE_WITHOUT_CONTRIBUTION,
+    UNKNOWN,
+    INELIGIBLE;
```

Comment:
This enum raises a few questions for me:

1. CMA can't give an indication on whether contributions are payable. We're currently working on another service that does that, but it won't be ready for some time. Personally, I think it would be more appropriate to simply have a `PASS`/`FAIL` result and leave any logic around contributions to CFE Crime.
2. Is the `UNKNOWN` state a placeholder, if not where do you see this being used?
3. Does `INELIGIBLE` simply denote a failed means assessment or are we referring to the specific `INEL` result that is triggered on full assessments when the disposable income threshold is exceeded?

In my mind, there are six possible outcomes from the means assessment calculation:

`INIT_PASS`, `INIT_FAIL`, `INIT_FULL`, `FULL_PASS`, `FULL_FAIL`, `FULL_INEL`

Reply:
I have been trying not to expose the PASS/FAIL/INEL to the client of this service. It feels like an internal implementation detail e.g. INIT_FULL is (I think) *not* the signal for the client to be able to perform a full assessment (its the isFullAssessmentAvailable boolean). Agree about the contribution thing though, that belongs upstream

Reply:
I'm not sure I agree that `PASS`/`FAIL`/`INEL` are internal implementation details. I agree on your suggestion around `INIT_FULL`, but in my mind the others are required. `MAAT` is also going to have to use this API in future and we need that additional information. Personally, I think we should just utilise the existing `InitAssessmentResult` and `FullAssessmentResult` enums from the existing API.

Reply:
I've gone for that option now - it's not ideal IMHO, but its probably a good compromise